### ISSUE TITLE
TimePanel error with moment format and type datetime

### ISSUE THREAD

**Issue by pippo46** [mengxiong10/vue2-datepicker#377]
**Vue2-datepicker version**: 3.0.2
**Vue version**: 2.6.10
**Browser**: all

DatePicker set with both type="datetime" and custom moment format raises the following error after selecting date and before opening TimePanel:

_[Vue warn]: Invalid prop: type check failed for prop "format". Expected String with value "[object Object]", got Object_

I don't get any errors when using format and type="date"

**Comment by mengxiong10** [comment]
Thanks for feedback.
It's a bug.
You need to set the `show-hour`  `show-minute` `show-second` now. Their default is false due to this bug.
I'll fix it, then the default value will be true and clear the warn.
```html
<date-picker
        v-model="value4"
        :format="momentForamt"
        :show-hour="true"
        :show-minute="true"
        :show-second="true"
        type="datetime"
        placeholder="Select datetime"
      ></date-picker>

```

**Comment by pippo46** [comment]
> 
> 
> Thanks for feedback.
> It's a bug.
> You need to set the `show-hour` `show-minute` `show-second` now. Their default is false due to this bug.
> I'll fix it, then the default value will be true and clear the warn.
> 
> ```
> <date-picker
>         v-model="value4"
>         :format="momentForamt"
>         :show-hour="true"
>         :show-minute="true"
>         :show-second="true"
>         type="datetime"
>         placeholder="Select datetime"
>       ></date-picker>
> ```

Thanks a lot for the quick reply, I'll try your solution.
In the meantime I also found this solution:

```
<DatePicker
  v-model="date"
  input-class="form-control"
  valueType="YYYY-MM-DDTHH:mm:ss"
  type="datetime" 
  :format="dpFormat" 
></DatePicker>`
```
```
import DatePicker from "vue2-datepicker";
export default Vue.extend({
  name: "home",
  components: {
    DatePicker
  },
  data() {
    return {
      dpFormat: `${moment.localeData().longDateFormat('L')} ${moment.localeData().longDateFormat('LT')}`
    };
  }
});
```

**Comment by mengxiong10** [comment]
fixed in v3.1.1
